spec/datasets/djornl/csv_edge.yaml

```diff
@@ -16,3 +16,10 @@ properties:
     pattern: ^\d+(\.\d+)?$
   edge_type:
     $ref: edge_type.yaml
+  directed:
```

Comment:
in the CSV input file, the directed column contains either `1` or `0` (shorter than writing out `true` or `false`, plus no need to worry about case)